src/components/FindWallet/WalletFilterSidebar/index.tsx

```diff
@@ -0,0 +1,265 @@
+import React, { MutableRefObject } from "react"
+import { BsToggleOff, BsToggleOn } from "react-icons/bs"
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  chakra,
+  GridItem,
+  HStack,
+  Icon,
+  List,
+  SimpleGrid,
+  Text,
+  useColorModeValue,
+  VStack,
+} from "@chakra-ui/react"
+import { uniqueId } from "lodash"
+import Checkbox from "../../Checkbox"
+import { useWalletFilterSidebar } from "./use-wallet-filter-sidebar"
+import { trackCustomEvent } from "../../../utils/matomo"
+
+const FilterToggle = ({
+  ariaLabel,
+  conditionItem,
+}: {
+  ariaLabel: string
+  conditionItem: boolean
+}) => (
+  <Box
+    as="span"
+    aria-label={ariaLabel}
+    role="checkbox"
+    aria-checked={conditionItem ? "true" : "false"}
+  >
+    <Icon
+      as={conditionItem ? BsToggleOn : BsToggleOff}
+      color="primary"
+      boxSize="1.875rem"
+    />
+  </Box>
+)
+
+export interface WalletFilterSidebarProps {
+  resetWalletFilter: MutableRefObject<() => void>
+  filters: Record<string, boolean>
+  updateFilterOption: (key: any) => void
+  updateFilterOptions: (key: any, value: any) => void
+}
+
+const WalletFilterSidebar: React.FC<WalletFilterSidebarProps> = ({
+  updateFilterOption,
+  ...restProps
+}) => {
+  const { filterOptions, setShowOptions } = useWalletFilterSidebar(restProps)
+
+  const filterPanelBg = useColorModeValue("primary100", "black400")
+  return (
+    <Accordion
+      as={VStack}
+      allowMultiple
+      reduceMotion
+      spacing={4}
+      alignItems="normal"
+      p={{ base: 4, sm: 0 }}
+      // Workaround to not having a dedicated prop to all items open by default
+      defaultIndex={Object.keys(filterOptions).map((key) => +key)}
+    >
+      {filterOptions.map((filterOption, idx) => {
+        return (
+          <AccordionItem
+            key={uniqueId("walletFilterSidebarItem")}
+            background={filterPanelBg}
+            borderRadius="base"
+            // Remove border color from global style
+            borderColor="transparent"
+            p={6}
+          >
+            {({ isExpanded }) => (
+              <>
+                <chakra.h3
```

Comment:
Curious about the difference in doing `<Text as="h3">` here.

Reply:
Nope, this was in error! 😄